clar2wasm/tests/wasm-generation/conditionals.rs

```diff
@@ -10,10 +11,10 @@ proptest! {
     #[test]
     fn if_true_returns_first_value(
         (v1, v2) in prop_signature()
-            .prop_flat_map(|ty| {
-                (PropValue::from_type(ty.clone()), PropValue::from_type(ty))
-            })
-        )
+        .prop_flat_map(|ty| {
+            (PropValue::from_type(ty.clone()), PropValue::from_type(ty))
+        })
+    )
```

Comment:
Why this change of formatting inside the macro? (and same for all following occurrences).

Reply:
rustfmt and/or my editor going wild, i could change it back but i'm a bit reluctant to rewrite history, if it's not bothering you i'll be more carful next time

Reply:
That's weird, it shouldn't touch the inside of macros :/